### ISSUE TITLE
See an edge?

### ISSUE THREAD

**Issue by o-jasper** [Jeija/spheretest#1]
I see an edge where i can see through, cannot mine through. Wether or not i can walk through as appropriate.

Suggested solution: empty the two planes of blocks at that, edge and then deform everything a bit more to close the gap? Seems like it might be.. not-so-hard? Just find the x,z coordinate to clear out, and change the deformation a bit?

Probably leaves at least the bug that adding/pushing blocks puts blocks into the gap. That should instead address the block past the gap...

Easier said than done ;) also this whole thing, i'd say, kudos for having done it :)

**Comment by Jeija** [comment]
Yes, this is a known issue. Collision detection and looking across the edge works fine, but drawing the faces of the nodes at the edge and selecting nodes doesn't.

I'd love to be able to fix these issues, but given I need to get other stuff done, it just isn't my priority right now, since spheretest is mostly intended as a proof-of-concept and not a blueprint for integrating sphere functionality into minetest. If someone is interested in fixing these issues or potentially upstreaming planet mode into minetest, I'd be glad to review and accept pull requests or give advice.

**Comment by Sam1370** [comment]
Can reproduce
